### PR TITLE
Note only VDAFs without aggregation-ordering requirement are supported.

### PR DESCRIPTION
**Pull request by branlwyd**

### CODE REVIEW

**Comment by branlwyd** [comment]
Part of #581.

File: draft-ietf-ppm-dap.md [hunk 1]
```diff
@@ -524,7 +524,8 @@ else about the measurements. We call `F` the "aggregation function".
 This protocol is extensible and allows for the addition of new cryptographic
 schemes that implement the VDAF interface specified in
 {{!VDAF=I-D.draft-irtf-cfrg-vdaf-12}}. This protocol only supports VDAFs which
-require a single collection to provide useful results.
+require a single collection to provide useful results, and whose aggregation
+results are independent of the order in which measurements are aggregated.
```
Comment (cjpatton) [inline draft-ietf-ppm-dap.md:528]: We should point to the relevant section of the VDAF draft. However we'll have to wait until VDAF-13 is out. Let's add a TODO to remind us, or file an issue?

**Review by cjpatton — APPROVED** [review]

**Review by tgeoghegan — APPROVED** [review]

**Comment by branlwyd** [comment]
Status: this PR is just waiting for VDAF-13 to be cut, at which point we can reference the appropriate section of the VDAF draft.

**Comment by branlwyd** [comment]
Rebased & squashed.

**Pull request merged**